mne/preprocessing/ssp.py

```diff
@@ -221,7 +224,7 @@ def _compute_exg_proj(mode, raw, raw_event, tmin, tmax,
     drop_log = epochs.drop_log
     if epochs.events.shape[0] < 1:
         warn('No good epochs found, returning None for projs')
-        return (None, events) + ((drop_log,) if return_drop_log else ())
+        return (None, epochs.events) + ((drop_log,) if return_drop_log else ())
```

Comment:
@larsoner which events should be returned here? The `events` from `find_ecg_events` or those from `create_ecg_epochs`? Shouldn't they be the same?

Reply:
They will only be the same if no trials are rejected, which in general will not be the case (e.g., if a trial is at the start or end of the raw instance, or reject/flat are provided).

Don't change which of these is returned, that is a backward compat break.

Reply:
First I thought they should not. Because we were testing `if epochs.events.shape[0] < 1` and then returning events. Then I thought they where the same Because otherwise why are we computing the Epochs if we return something that we already had before computing the Epochs